client/post-editor/editor-action-bar/index.jsx

```diff
@@ -72,7 +72,7 @@ export default React.createClass( {
 				</div>
 				<EditorPostType />
 				<div className="editor-action-bar__last-group">
-					{ this.props.post && this.props.type === 'post' && <EditorSticky post={ this.props.post } /> }
+					{ this.props.post && this.props.type === 'post' && <EditorSticky /> }
```

Comment:
For a future pull request, I think it would be nice for this logic to be contained within `<EditorSticky />` so it's self-sufficient. In other words, it could return `null` if the edited post is not of type `post`.